hazelcast/src/main/java/com/hazelcast/partition/impl/InternalPartitionServiceImpl.java

```diff
@@ -1677,11 +1677,7 @@ public void dispatchEvent(PartitionEvent partitionEvent, PartitionEventListener
 
     @Override
     public String toString() {
-        StringBuilder sb = new StringBuilder("PartitionManager[" + stateVersion + "] {\n");
-        sb.append("\n");
-        sb.append("migrationQ: ").append(migrationQueue.size());
-        sb.append("\n}");
-        return sb.toString();
+        return "PartitionManager[" + stateVersion + "] {\n" + "\n" + "migrationQ: " + migrationQueue.size() + "\n}";
```

Comment:
I have merged the enters and I don't know why there are so many enters. But lets make that part of a different pr.